Models/Student.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace StudentService.Models
+{
+    public class Student
+    {
+        [Key]
+        public int Id { get; set; }
+        [Required]
+        public string FirstName { get; set; }
+        [Required]
+        public string LastName { get; set; }
```

Comment:
ain't you supposed to use camel case naming conventions for variable name

Reply:
These are properties and properties name are PascalCase in C#